tests/CMakeLists.txt

```diff
@@ -36,6 +36,13 @@ add_test(
 )
 add_dependencies(check unittests)
 
+# FIXME: REMOVE THIS
+# Installing unittests to check installation of ortools
+install(TARGETS unittests
+  LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+  ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+  RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
+)
```

Comment:
Running the unittests from the install directory worked for me (tm)

Reply:
Also for me